src/Payum/Paypal/ExpressCheckout/Nvp/Tests/Action/Api/CancelRecurringPaymentsProfileActionTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+namespace Payum\Paypal\ExpressCheckout\Nvp\Tests\Action\Api;
+
+use Payum\Core\Request\Cancel;
+use Payum\Paypal\ExpressCheckout\Nvp\Action\Api\CancelRecurringPaymentsProfileAction;
+use Payum\Paypal\ExpressCheckout\Nvp\Request\Api\ManageRecurringPaymentsProfileStatus;
+
+class CancelRecurringPaymentsProfileActionTest extends \PHPUnit_Framework_TestCase
```

Comment:
Try to extend from https://github.com/Payum/Payum/blob/master/src/Payum/Core/Tests/GenericActionTest.php class. and configure providers methods.